socorro/unittest/external/boto/conftest.py

```diff
@@ -0,0 +1,45 @@
+import boto
+from boto.exception import StorageResponseError
+import pytest
+
+
+class BotoHelper(object):
+    """Helper class inspired by Boto's S3 API
+
+    The goal here is to automate repetitive things in a convenient way, but
+    be inspired by the existing Boto S3 API.
+
+    """
+    def get_or_create_bucket(self, bucket_name):
+        """Gets or creates the crashstats bucket"""
+        conn = boto.connect_s3()
+        try:
+            bucket = conn.get_bucket(bucket_name)
+        except StorageResponseError:
+            conn.create_bucket(bucket_name)
+            bucket = conn.get_bucket(bucket_name)
+        return bucket
+
+    def set_contents_from_string(self, bucket_name, key, value):
+        """Puts an object into the specified bucket"""
+        bucket = self.get_or_create_bucket(bucket_name)
+        key_object = bucket.new_key(key)
+        key_object.set_contents_from_string(value)
+
+    def get_contents_as_string(self, bucket_name, key):
+        """Fetches an object from the specified bucket"""
+        bucket = self.get_or_create_bucket(bucket_name)
+        key_object = bucket.get_key(key)
+        if key_object is None:
+            return None
+        return key_object.get_contents_as_string()
+
+    def list(self, bucket_name):
+        """Lists contents of bucket and returns keys as strings"""
+        bucket = self.get_or_create_bucket(bucket_name)
+        return [key.key for key in bucket.list()]
+
+
+@pytest.fixture
+def boto_helper():
+    return BotoHelper()
```

Comment:
That makes sense. I tend to make things fixtures that are part of the test infrastructure because it makes it easier to re-use elsewhere and it's easier to discover in large test suites (`pytest --fixtures`). Oops--I should add a docstring.